pkg/minikube/machine/fix.go

```diff
@@ -170,9 +170,9 @@ func maybeWarnAboutEvalEnv(drver string, name string) {
 	if p == "" {
 		return
 	}
-	out.T(out.Notice, "Noticed that you are using minikube docker-env:")
-	out.T(out.Warning, `After minikube restart the dockerd ports might have changed. To ensure docker-env works properly.
-Please re-eval the docker-env command:
+	out.T(out.Notice, "Noticed you have an activated docker-env on {{.driver_name}} driver in this terminal:", out.V{"driver_name": drver})
```

Comment:
I would change `Noticed` to `It seems like you have an activated docker-env ... `